SupportBank/Program.cs

```diff
@@ -1,12 +1,46 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using CsvHelper;
 
 namespace SupportBank
 {
     class Program
     {
+        private static Dictionary<string,Account> _accounts;
+        
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            _accounts = new Dictionary<string, Account>();
+            CSVReader();
+            foreach (var account in _accounts.Values)
+            {
+                Console.WriteLine($"Name: {account.Name} -- Pending Account Balance: {account.BalanceDue}");
+            }
+        }
+
+        static void CSVReader()
+        {
+            using (var reader = new StreamReader("./Transactions2014.csv"))
+            using (var csv = new CsvReader(reader, CultureInfo.InvariantCulture))
```

Comment:
With C# 8 you don't need to have the brackets with the using statement: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement 🙂